src/models/linear_gaussian_conditionals.jl

```diff
@@ -28,10 +34,11 @@ Base.:(==)(x::AbstractLGC, y::AbstractLGC) = (x.A == y.A) && (x.a == y.a) && (x.
 Base.eltype(f::AbstractLGC) = eltype(f.A)
 
 """
-    predict(x::Gaussian, f::AbstractLGC)
+    predict(x::Gaussian, f::AbstractLGC)::Gaussian{Tm,Matrix}
```

Comment:
```suggestion
    predict(x::Gaussian, f::AbstractLGC)::Gaussian{Tm,AbstractMatrix}
```
Probably, because static arrays